nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -403,6 +409,44 @@ in {
         };
       };
 
+      aws = {
+        SecretAccessKeyFile = lib.mkOption {
+          description = lib.mdDoc ''
+            Path to file containing the SecretAccessKeyFile.
+          '';
+          default = "/var/lib/mastodon/secrets/mastodon-secret-access-key";
+          example = "/run/keys/mastodon-smtp-password";
```

Comment:
There isn't and can't be an option to auto-generate this file, so having a default is not useful. The example is also still smtp.

I would delete the default and update the example.